roles/agnosticv/templates/catalog_item.yml.j2

```diff
@@ -16,6 +16,9 @@ metadata:
 {% if _total_ratings_annotation != '' %}
     babylon.gpte.redhat.com/totalRatings: {{ _total_ratings_annotation | to_json }}
 {% endif %}
+{% if _last_successful_provision_annotation != '' %}
+    babylon.gpte.redhat.com/lastSuccessfulProvision: {{ _last_successful_provision_annotation | to_json }}
+{% endif %}
```

Comment:
What sets `babylon.gpte.redhat.com/lastSuccessfulProvision`?

Reply:
Is a timestamp that the catalog-manager operator sets by querying the reporting db